sqlglot/parser.py

```diff
@@ -8244,3 +8244,11 @@ def _parse_max_min_by(self, expr_type: t.Type[exp.AggFunc]) -> exp.AggFunc:
         return self.expression(
             expr_type, this=seq_get(args, 0), expression=seq_get(args, 1), count=seq_get(args, 2)
         )
+
+    def _identifier_expression(
+        self, token: t.Optional[Token] = None, **kwargs: t.Any
+    ) -> exp.Identifier:
+        token = token or self._prev
+        expression = self.expression(exp.Identifier, this=token.text, **kwargs)
+        expression.meta["token"] = token
```

Comment:
Should we store the attributes we care about instead of the whole token? Also, what about making these `Expression` attributes that default to `None` instead of putting them in `meta`? Don't feel strongly about this, just feel like we've been doing this for other attributes, like `type`.

cc @tobymao

Reply:
Discussed internally:

- Remove raw token object, store its attributes directly instead (to avoid issues with Rust tokens etc)
- Go with the `meta` approach.